libs/sdk-ui-web-components/nginx.conf

```diff
@@ -0,0 +1,65 @@
+# (C) 2021 GoodData Corporation
+worker_processes  auto;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /tmp/nginx.pid;
+
+events {
+  worker_connections  1024;
+}
+
+http {
+  server {
+    listen 8080;
```

Comment:
@mouchar , I left the default port, but I think we'll need to change it for the AIO image.

Reply:
AIO has its own nginx, so this file is completely irrelevant to AIO build. Having 8080 port might be an issue for docker-compose deployment. We try to avoid these well-known default ports and set our own where is a lower chance of port conflicts.
On the other hand, you can keep 8080 as is and we can map this port to a different value in in docker-compose, like:
```yaml
web-components:
  ...
  ports:
  - 9800:8080
```